product_development.md

```diff
@@ -0,0 +1,96 @@
+
+All work that is done in either product, design or engineering should be reflected on the board. Stakeholders may have tickets on the board too.
+
+### New feature tickets
+
+The requirements engineering and conception of new features is owned by the product and design team.
+
+### Growth opportunity
+
+Anything that can help a department (not only product related) to improve their performance and the performance of our business.
+
+Growth opportunities should align with our long term product vision.
+
+Depending on the complexity those opportunities might be developed in small increments prioritizing value delivery.
+
+### OKR initiatives
+
+Strategic initiatives developed with a department and prioritized in close coordination with an objective owner. Those initiatives could be features and also research tasks, surveys, content work and more.
+
+They are prioritized over anything else to ensure the success of our OKRs.
+
+OKR initiatives should be in line with our long-term product vision.
+
+### Operational hiccups
+
+Operational hiccups are issues that disturb a stakeholders operation or hinder their ability to exercise their responsibilities. They usually require some kind of technical support or facilitation and should have some level of urgency similar to bugs but are outside of the engineering responsibilities.
+
+### Bugs
+
+Bugs are owned by the engineering team as they affect the platform's stability.
+
+Bugs must have some kind of urgency. They are not prioritized by product, but triaged by engineering, based on what bug will kill our business first. Triage is handled by a designated senior engineer. Bugs are assigned to an engineer at all times but can be reassigned.
+
+A bug ticket should have an average resolution rate of less than a week.
+
+Bugs may be closed or converted into a story, if there is no longer a high urgency and no impact on the stability of the platform. Updates on bug tickets, especially closure without resolution, need to be communicated with product proactively.
+
+### Tech debt
+
+Engineering leads are responsible for the stability of the platform and the development velocity, and consequently for when and what technical debt needs to be handled.
+
+Work on technical debt must always be proactively communicated with the product team and documented in a ticket. Tech debt tickets must be pulled into the backlog in a refinement meeting, as they bind resources and take up significant space in the engineering backlog.
+
+### Platform announcement Slack channel
+
+Product, Design, Engineering, and Content team members post every change to go live on the platform into the Slack channel [#platform-announcements](https://voiio.slack.com/archives/C02BXK6R16V). This channel serves as a changelog for all employees.
```

Comment:
I think this should be more precise. If there are changes to the platform (say we are improving the performance of a background sync job) we tend to not share this because there is no value in knowing this for anybody but the dev team and may make the channel too noisy.

Reply:
This is working very well right now, I don't understand the confusion. Just keep doing, what you are doing and inform voiio of all changes to the platform experience.